src/component/ExerciseInfo.tsx

```diff
@@ -48,7 +48,9 @@ export const FilterMuscleGroup = ({
 
 export const ExerciseDetails = ({item}: {item: SequenceItem}) => {
   const filterText = (text: string) => {
-    return text.replace(/<[^>]*>?/gm, '');
+    return text
+      .replace(/<[^>]*(>|$)|&nbsp;|&zwnj;|&raquo;|&laquo;|&gt;/g, '')
+      .trim();
```

Comment:
this seems very hacky, i'd try to render HTML text instead. Something like: https://blog.logrocket.com/how-render-html-react-native/

Reply:
I think that might be overkill for now. let's ignore it